### PR TITLE
MigratingAuthors

### PR DESCRIPTION
**Pull request by simgobb**
Added all the authors of the articles labelled as Web.dev in the ContentMigration Sheet

@rachelandrew

### CODE REVIEW

**Comment by rachelandrew** [comment]
@simgobb this is web.dev, you mentioned DCC. Can you confirm these are authors for web.dev articles not DCC?

**Comment by simgobb** [comment]
@rachelandrew Yes, sorry. I confirm that these are the Web Dev ones I got from the migration sheet. 

I don't think we have any content ready yet as we focused on DCC, but I thought it would be good to sort this side of things as well to avoid the issues we had with DCC.

**Comment by rachelandrew** [comment]
@simgobb it looks like these are going to need images. I think that's why it is not building. You can upload the ones used on WebFundamentals.

File: src/site/_data/i18n/authors.yml [hunk 1]
```diff
@@ -1,3 +1,81 @@
...
+  title:
+    en: Dale Curtis
+  description:
+    en: Dale is a Senior Software Engineer
+
+mscales:
+  title:
+    en: Mat Scales
+  description:
+    en: Mat is a contributor to WebFundamentals
```
Comment (rachelandrew) [inline src/site/_data/i18n/authors.yml:17]:
```suggestion
    en: Mat is a contributor to web.dev
```

File: src/site/_data/i18n/authors.yml [hunk 2]
```diff
@@ -1,3 +1,81 @@
...
+
+ewagasperowicz:
+  title:
+    en: Ewa Gasperowicz
+  description:
+    en: Ewa is a contributor to WebFundamentals
```
Comment (rachelandrew) [inline src/site/_data/i18n/authors.yml:23]:
```suggestion
    en: Ewa is a contributor to web.dev
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
I just realised that a lot of these have a bio of "... is a contributor to WebFundamentals" can we update all of these to web.dev?

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**